migrations/20200427161609-add-body-and-description-to-position.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    up: (queryInterface, Sequelize) => queryInterface.addColumn(
+        'positions',
+        'body_id',
+        { type: Sequelize.INTEGER, allowNull: true }
+    ),
+    down: queryInterface => queryInterface.removeColumn('postiions', 'body')
```

Comment:
```suggestion
    down: queryInterface => queryInterface.removeColumn('positions', 'body')
```
`down` does not match `up` here